### ISSUE TITLE
Erratic scrolling with macbook pro track pad (two finger scrolling)

### ISSUE THREAD

**Issue by GLBaird** [Komodo/KomodoEdit#190]
When using on-screen controls to scroll window, scrolling is smooth and fine, however, when doing the normal two finger scroll using a mac book pro trackpad or external magic trackpad from Apple, the scrolling is erratic and jumps around and sticks, and can be very frustrating to move around the document. This was never an issue with older Komo Edit or Komodo IDE.

**Comment by cgchoffman** [comment]
What version of Komodo are you currently using? Help > About Komodo.  Please include the build number.
What was the actual version of the older Komodo that works?
Do you have Mini-map enable for the view that you're scolling? View > View Minimap

**Comment by GLBaird** [comment]
I was using Komodo IDE 9.0, I sadly cannot include the build number, as the trial version has expired and I am not going to purchase it unless it becomes more stable, as I have been thinking for a while to migrate to Komodo IDE from Edit. But the same occurs in Komodo Edit, which I am currently using instead of the IDE version:

Komodo Edit, version 9.0.0, build 15707

It seems to be problematic with very large code files more than short files. It becomes so hard to scroll over time, especially as Komodo can stay launched for several days/weeks at once on my system, that I have gone back to Komodo Edit 8 for the time being. 

I have many JS/PHP files of 9000+ lines of code, and when line numbers are being displayed (or not, but worse with), the new way it seems to scroll text, the way it redraws the line numbers, seems to get slower with time and even from the start is very sluggish, and it starts to feel as if the computer maxes out performance wise, and becomes hard to control smoothly as usual it does.

Never had this problem with earlier version of Komodo Edit. Version used: Komodo Edit version 8.5.4, build 14424. I guess the drawing code is not as optimised as earlier versions and may have some memory leaks. You do not feel it when scrolling with the scroll bars or mouse, but as the track pad on Macs are very fast and very smooth - it can rush through hundreds of lines of code in a split second, it feels uncomfortable and becomes so annoying to control scrolling that I am going to wait before trying again.

I am using a Macbook Pro  Late 2013, 2.6 GHz Intel Core i5 with 8GB Ram. OS X10.10.1.

Best wishes,
Leon Baird

> On 20

**Comment by Naatan** [comment]
Could you try;

 - Plugging in a regular mouse and see if it affects trackpad scrolling
 - Let us know if the issue occurs when scrolling with a regular mouse
 - Try disabling "Scrollbar Skinning" under `Prefs > Appearance`

**Comment by Naatan** [comment]
Closing due to inactivity.

**Comment by dataxpress** [comment]
I have this same issue on KomooEdit 10.1.4 build 17456, Mac OS 10.12.2 MacBook Pro 2.3 GHz / 16 GB. Scrolling is choppy even on an HTML file with only 355 lines. I have tried disabling as many editor features as possible, including line numbers, code linting, and scroll map. Using laptop trackpad or bluetooth Magic Trackpad makes no difference. Very fine movements scrolling cause the buffer to "twitch" but not actually scroll - seems to be the editor trying to lock the scroll view to whole lines. When I scroll faster, the line numbers do not stay "attached" to their respective lines until scrolling is finished, as if the line number gutter is a few frames behind the main buffer. I used KomodoEdit back in the day and was hoping to use it again after giving up on Atom but these issues make it a headache to use. 

Is there any way to disable scroll snapping to lines? I could not find anything in Settings.
